src/App.vue

```diff
@@ -35,6 +38,11 @@ export default {
         breadcrumbsRoutes.includes(this.$route.name)
       );
     },
+    // booking pages: choose seats, choose tickets
+    showStepsNav() {
+      const stepsNavRoutes = ["ChooseSeats", "ChooseTickets"];
+      return stepsNavRoutes.includes(this.$route.name);
+    },
```

Comment:
Seems like you're using computed values to determine which Header should be shown. 
One problem I can see going forward is that the logic might at some point become too complicated to determine at a glance which Header/NavBar should be visible.

I would suggest adding Headers directly in the ChooseSeats/ChooseTickets pages components, this way you won't have to think which Header should be visible at the current moment.  At that point, you could remove also all logic from App.vue and leave only <router-view /> and let each page handle its Headers on its own.